apps/examples/camera/camera_main.c

```diff
@@ -449,7 +449,25 @@ void get_camera_options(int fd, int flag)
 		fprintf(stdout, SET_INTERVAL);
 	}
 }
+void print_error_log(int ctrlIndex)
+{
+	char error_str[50] = {0,};
+	int idx = 0;
+
+	while (idx < 50) {
+		error_str[idx] = ctrls[ctrlIndex].sbuff[idx];
+		if (ctrls[ctrlIndex].sbuff[idx] == ']')
+			break;
+		idx++;
+	}
 
+	if (errno == EINVAL || errno == -EINVAL) {
```

Comment:
please clean it. use negative only or positive only. please check errno logic.

Reply:
Done